addons/website_slides/static/src/js/public/components/slide_upload_dialog/slide_upload_source_types.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import { Component, useState } from "@odoo/owl";
+
+export class SlideUploadSourceTypes extends Component {
+    static props = {
+        attributes: {
+            type: Object,
+            shape: {
+                sourceTypeLabel: String,
+                selectFileLabel: String,
+                acceptedFiles: String,
+                urlInputLabel: String,
+                urlInputName: String,
+            },
+        },
+        isLocalSource: Boolean,
+        onClickSourceType: Function,
+        onChangeFileInput: Function,
+        onChangeUrl: Function,
+    };
+    static template = "website_slides.UploadSourceTypes";
```

Comment:
can we use the same name for the template and the JS class ? (easier to grep)

double check other templates / classes